app/domain/authentication/authn_jwt/consts.rb

```diff
@@ -20,11 +20,15 @@ module AuthnJwt
     EXP_CLAIM_NAME = "exp"
     NBF_CLAIM_NAME = "nbf"
     IAT_CLAIM_NAME = "iat"
+    JTI_CLAIM_NAME = "jti"
     SUPPORTED_ALGORITHMS = %w[RS256 RS384 RS512].freeze
     CACHE_REFRESHES_PER_INTERVAL = 10
     CACHE_RATE_LIMIT_INTERVAL = 300
     CACHE_MAX_CONCURRENT_REQUESTS = 3
     MANDATORY_CLAIMS = [EXP_CLAIM_NAME].freeze
     OPTIONAL_CLAIMS = [ISS_CLAIM_NAME, NBF_CLAIM_NAME, IAT_CLAIM_NAME].freeze
+    VALID_CLAIM_NAME_REGEX = /^[a-zA-Z|$|_][a-zA-Z|$|_|0-9]*$/ # starts with letter $ or _ can contains digits
+    MANDATORY_CLAIMS_RESOURCE_NAME = "mandatory-claims"
+    MANDATORY_CLAIMS_DENY_LIST = [ISS_CLAIM_NAME, EXP_CLAIM_NAME, NBF_CLAIM_NAME, IAT_CLAIM_NAME, JTI_CLAIM_NAME].freeze
```

Comment:
@nessiLahav what should we do with other public claims like : auth_time (https://www.iana.org/assignments/jwt/jwt.xhtml) we will allow it?

Reply:
@benoaviram good question

@sashaCher @hilagross WDYT?

Reply:
No limitations from my point of view. May be we need to put some general disclaimer in doc that time valued claims could not be annotations, if it's not obvious.

Reply:
@nessiLahav Please add `aud` to the `DENY_LIST`.

Reply:
Agree with @sashaCher I would have add it to our documentation task now so we won't forget it.

Reply:
@sashaCher why adding aud? if the customer will want to enforce aud only to one instance of authenticator? (without using the variable)

Reply:
As long as `aud` is enforced on authenticator's level (like `iss` ) adding it to mandatory and/or mapping is useless.